addons/map_gestures/ACE_Settings.hpp

```diff
@@ -0,0 +1,50 @@
+class ACE_Settings {
+    class GVAR(enabled) {
+        value = 1;
+        typeName = "BOOL";
+        displayName = CSTRING(enabled_displayName);
+        description = CSTRING(enabled_description);
+    };
+    class GVAR(maxRange) {
+        value = 7;
+        typeName = "SCALAR";
+        displayName = CSTRING(maxRange_displayName);
+        description = CSTRING(maxRange_description);
+    };
+    class GVAR(interval) {
+        value = 0.03;
+        typeName = "SCALAR";
+        displayName = CSTRING(interval_displayName);
+        description = CSTRING(interval_description);
+    };
+    class GVAR(defaultLeadColor) {
```

Comment:
I get that. But why? Colors are imo a user specific setting. It would help color blind people to set the color to something they can see clearly. What is the benefit of having this set globaly or the reason behind it?